test/services/pin/post.test.js

```diff
@@ -264,6 +264,9 @@ describe('Pin - POST', () => {
       });
   });
 
+  // TODO: Add a test case if a user creates pin without default.location in config file
```

Comment:
We should also add a test before the app starts and warns user to add some compulsory config.